### DISCUSSION TITLE
Allow "peeking" print statement hits

### DISCUSSION THREAD

**Discussion by jcmorrow** [RecordReplay/devtools#7148] [Ideas]
I just discovered Linear's "peek" feature, which allows you to hit space bar while hovering a list of issues and get a quick preview. I think we could do something amazing with a similar feature for print statements, where you press a key while hovering a print statement panel and moving over points will you show you a hovered window that includes the value of the print statement and maybe some basic pause data as well?

https://user-images.githubusercontent.com/5903784/173348497-fe05ea18-2f86-4960-9790-3df109662f2f.mp4

I just really love how easy it is to engage/dismiss this, I think there's something good here.

**Comment by jonbell-lot23** [comment]
I love this!

One difference between us and Linear is they always have a solid hit target, that's the same size, with a floating window that's always the same size. Whereas our hit targets are small and sometimes bunched.

Still a great idea, and I think it could be really good. But it reminds me that having a zoom/scale mechanism would really help too. To make things less bunched, especially on a long replay.

**Comment by jcmorrow** [comment]
I just got the email notification about your comment and reread it and was like "wow, that sounds like a great idea!" Only to realize I suggested this and then promptly forgot it 😆
